download_collection.py

```diff
@@ -0,0 +1,722 @@
+# Script to download Scanned Objects by Google Research dataset and Stanford models
+# Distributed by CC-BY 4.0 License
+
+# Dataset page:
+# https://app.gazebosim.org/GoogleResearch/fuel/collections/Scanned%20Objects%20by%20Google%20Research
+
+import sys, argparse, subprocess
+from pathlib import Path
+import requests
+import zipfile, tarfile, gzip
+import csv, json
+import math
+import numpy as np
+
+try:
+    import cv2 as cv
+except ImportError:
+    raise ImportError("Failed to import OpenCV, please check that cv2 library is in PYTHONPATH env variable, "
+                      "for example use the path {opencv_build_dir}/lib/python3/")
+
+if sys.version_info[0] < 3 or sys.version_info[1] < 5:
+    raise Exception("Python 3.5 or greater is required. Try running `python3 download_collection.py`")
```

Comment:
Please use `Version` for it: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python